src/components/Viewport.ts

```diff
@@ -395,6 +396,11 @@ export default class Viewport {
 
     this.resize();
 
+    const isFreeScroll = (this.options.moveType as MoveTypeObjectOption).type === "freeScroll";
+    if (isFreeScroll && wasEmpty) {
+      this.moveTo(this.currentPanel!, "", null, 0);
```

Comment:
@WoodNeck Is above code related with `resizing`?

Reply:
Nope, it is related with adding new panel when there were no panels in free scroll mode.
As free scroll maintains camera position while in resizing, camera position usually is wrong after adding new panel.
So, I've manually modified it for its circumstance.